### ISSUE TITLE
Add !gimage trigger

### ISSUE THREAD

**Issue by adamstirtan** [adamstirtan/ircbot#2]
URL: https://ajax.googleapis.com/ajax/services/search/images?v=1.0&q=Paris%20Hilton

Response:

{"responseData": {"results":[{"GsearchResultClass":"GimageSearch","width":"1216","height":"1096","imageId":"ANd9GcSfsedMmgXvlGXSJoCo-A17i92s_smrjZIvSpy4WNrAEVax_9KqOTNrUhx-AQ","tbWidth":"150","tbHeight":"135","unescapedUrl":"https://upload.wikimedia.org/wikipedia/commons/2/26/Paris_Hilton_3_Crop.jpg","url":"https://upload.wikimedia.org/wikipedia/commons/2/26/Paris_Hilton_3_Crop.jpg","visibleUrl":"en.wikipedia.org","title":"\u003cb\u003eParis Hilton\u003c/b\u003e - Wikipedia, the free encyclopedia","titleNoFormatting":"Paris Hilton - Wikipedia, the free encyclopedia","originalContextUrl":"https://en.wikipedia.org/wiki/Paris_Hilton","content":"\u003cb\u003eHilton\u003c/b\u003e in 2006, when she was","contentNoFormatting":"Hilton in 2006, when she was","tbUrl":"http://t1.gstatic.com/images?q\u003dtbn:ANd9GcSfsedMmgXvlGXSJoCo-A17i92s_smrjZIvSpy4WNrAEVax_9KqOTNrUhx-AQ"},{"GsearchResultClass":"GimageSearch","width":"300","height":"400","imageId":"ANd9GcSx5x9vozSCuqMXCDXRjQv2TZsiyqQLb17gU-6RbGaKR0GyatUA8ikjUaY","tbWidth":"93","tbHeight":"124","unescapedUrl":"http://media1.popsugar-assets.com/files/2012/12/51/4/192/1922398/592a846386cc66b4_parishilton.xxxlarge_2.jpg","url":"http://media1.popsugar-assets.com/files/2012/12/51/4/192/1922398/592a846386cc66b4_parishilton.xxxlarge_2.jpg","visibleUrl":"www.popsugar.com","title":"\u003cb\u003eParis Hilton\u003c/b\u003e | POPSUGAR Celebrity","titleNoFormatting":"Paris Hilton | POPSUGAR Celebrity","originalContextUrl":"http://www.popsugar.com/Paris-Hilton","content":"\u003cb\u003eParis Hilton\u003c/b\u003e","contentNoFormatting":"Paris Hilton","tbUrl":"http://t1.gstatic.com/images?q\u003dtbn:ANd9GcSx5x9vozSCuqMXCDXRjQv2TZsiyqQLb17gU-6RbGaKR0GyatUA8ikjUaY"},{"GsearchResultClass":"GimageSearch","width":"2000","height":"1000","imageId":"ANd9GcTy61ABBfjolrAV7YvKlgjP-Rx_C0JF-4IECpfyQfp9zjtN8nqiUlw8NCMF","tbWidth":"150","tbHeight":"75","unescapedUrl":"http://www.aux.tv/wp-content/uploads/2015/06/paris-hilton-diancam-akan-dibunuh.jpg","url":"http://www.aux.tv/wp-content/uploads/2015/06/paris-hilton-diancam-akan-dibunuh.jpg","visibleUrl":"www.aux.tv","title":"\u003cb\u003eParis Hilton\u003c/b\u003e might be a vintage radio enthusiast and a nice person \u003cb\u003e...\u003c/b\u003e","titleNoFormatting":"Paris Hilton might be a vintage radio enthusiast and a nice person ...","originalContextUrl":"http://www.aux.tv/2015/06/paris-hilton-might-be-a-vintage-radio-enthusiast-and-a-nice-person/","content":"\u003cb\u003eParis Hilton\u003c/b\u003e might be a","contentNoFormatting":"Paris Hilton might be a","tbUrl":"http://t2.gstatic.com/images?q\u003dtbn:ANd9GcTy61ABBfjolrAV7YvKlgjP-Rx_C0JF-4IECpfyQfp9zjtN8nqiUlw8NCMF"},{"GsearchResultClass":"GimageSearch","width":"630","height":"472","imageId":"ANd9GcSDiiwe048OdgY6-qNtW30xPW4jp18_dhMr5jajt0YkvkCdSPHobU7yj1w","tbWidth":"137","tbHeight":"103","unescapedUrl":"http://static3.businessinsider.com/image/5256df2eeab8eab502bb724d/paris-hilton-lashes-out-at-radio-host-after-thinking-shes-off-air.jpg","url":"http://static3.businessinsider.com/image/5256df2eeab8eab502bb724d/paris-hilton-lashes-out-at-radio-host-after-thinking-shes-off-air.jpg","visibleUrl":"www.businessinsider.com","title":"\u003cb\u003eParis Hilton\u003c/b\u003e Lashes Out At Radio Host After Thinking She\u0026#39;s Off-Air \u003cb\u003e...\u003c/b\u003e","titleNoFormatting":"Paris Hilton Lashes Out At Radio Host After Thinking She\u0026#39;s Off-Air ...","originalContextUrl":"http://www.businessinsider.com/paris-hilton-lashes-out-at-radio-host-after-thinking-shes-off-air-2013-10","content":"\u003cb\u003eParis Hilton\u003c/b\u003e Lashes Out At","contentNoFormatting":"Paris Hilton Lashes Out At","tbUrl":"http://t0.gstatic.com/images?q\u003dtbn:ANd9GcSDiiwe048OdgY6-qNtW30xPW4jp18_dhMr5jajt0YkvkCdSPHobU7yj1w"}],"cursor":{"resultCount":"9,730,000","pages":[{"start":"0","label":1},{"start":"4","label":2},{"start":"8","label":3},{"start":"12","label":4},{"start":"16","label":5},{"start":"20","label":6},{"start":"24","label":7},{"start":"28","label":8}],"estimatedResultCount":"9730000","currentPageIndex":0,"moreResultsUrl":"http://www.google.com/images?oe\u003dutf8\u0026ie\u003dutf8\u0026source\u003duds\u0026start\u003d0\u0026hl\u003den\u0026q\u003dParis+Hilton","searchResultTime":"0.17"}}, "responseDetails": null, "responseStatus": 200}
